.github/workflows/community.yml

```diff
@@ -26,7 +27,7 @@ jobs:
           - name: postgres
           - name: search
           - name: tomcat
-          - name: transformers
+#          - name: transformers
```

Comment:
in this way you are also excluding non-rocky tests for this role, you could use the `exclude` directive https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations